sdk/state/state.go

```diff
@@ -22,7 +22,6 @@ type Amount struct {
 type EscrowAccount struct {
 	Address        *keypair.FromAddress
 	SequenceNumber int64
-	Balances       []Amount
```

Comment:
this field isn't used atm. It may be used when we support multiple assets, but for now removing